Updating License Information Documentation (#4019)
mattermost/docs: source/install/ee-install.rst

.. note::

  If you upload the license via the CLI using  ``mattermost license upload``, you need to restart the Mattermost server after uploading. Additionally, if you're running a cluster, the license file needs to be uploaded to every node. See `our documentation for more information on the command line tools <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-license-upload>`__.
  
License key storage

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Reviewer: ```suggestion
^^^^^^^^^^^^^^^^^^^^
```

> Once you have uploaded your license key to your Mattermost server it will be stored in your SQL database at `mattermost.Licenses`. You can check what keys are on your server by running `select * from mattermost.Licenses;`.

Reviewer: ```suggestion
Once you've uploaded your license key to your Mattermost server it's stored in your SQL database at ``mattermost.Licenses``. You can check what keys are on your server by running ``select * from mattermost.Licenses;``.
```

**Using the System Console**

1. Open **System Console > About > Edition and License** (or **System Console > OTHER > Edition and License** in versions prior to 5.12).
2. Click **Remove Enterprise License and Downgrade Server**. This clears the license from the server and refreshes the System Console.
3. Upload the new license key file.

> Removing the license key to upgrade will not remove settings that you have configured with an Enterprise license so long as your new license key is the same or higher level license. 

Reviewer: ```suggestion
Removing the license key to upgrade will not remove any Enterprise settings that you have configured as long as your new license key is the same or higher level license.
```

> Removing the license key to upgrade will not remove any Enterprise settings that you have configured as long as your new license key matches the previous version of Enterprise Edition, or is for a higher version.

Reviewer: ```suggestion
Removing an E20 or E10 license key will not remove the configuration for Enterprise settings, however these features will not function until an E10 or E20 license key is applied. 
.. note::
  - When you apply an E20 license key to a previously E10-licensed server, the E10 features will retain their configuration settings in E20. 
  - When you apply an E10 license to a previously E20-licensed server, the E20 features will retain their configuration but will no longer be accessible for use.
```

  
License key storage
^^^^^^^^^^^^^^^^^^^^

Once you've uploaded your license key to your Mattermost server it's stored in your SQL database at ``mattermost.Licenses``. You can check what keys are on your server by running ``select * from mattermost.Licenses;``.

**Using the System Console**

1. Open **System Console > About > Edition and License** (or **System Console > OTHER > Edition and License** in versions prior to 5.12).
2. Click **Remove Enterprise License and Downgrade Server**. This clears the license from the server and refreshes the System Console.
3. Upload the new license key file.

Removing an E20 or E10 license key will not remove the configuration for Enterprise settings, however these features will not function until an E10 or E20 license key is applied. 

.. note::

  - When you apply an E20 license key to a previously E10-licensed server, the E10 features will retain their configuration settings in E20. 
  - When you apply an E10 license to a previously E20-licensed server, the E20 features will retain their configuration but will no longer be accessible for use.

> 

Reviewer 2: Please add:  Please note that when using High Availability to ensure all servers in the cluster have the E20 license properly installed to prevent multi-node clusters from failing. An E20 license is required for High Availability to work.

Maybe this can go in at line 197.

Reviewer: I've added it and agree with the placement.
